lib/Feature/FeatureModelParser.cpp

```diff
@@ -33,7 +33,28 @@ bool FeatureModelXmlParser::parseConfigurationOption(xmlNode *Node,
             if (!xmlStrcmp(Child->name, XmlConstants::OPTIONS)) {
               std::unique_ptr<xmlChar, void (*)(void *)> CCnt(
                   xmlNodeGetContent(Child), xmlFree);
-              FMB.addExclude(Name, reinterpret_cast<char *>(CCnt.get()));
+              FMB.addConstraint(std::make_unique<ImpliesConstraint>(
+                  std::make_unique<PrimaryFeatureConstraint>(
+                      std::make_unique<Feature>(Name)),
+                  std::make_unique<NotConstraint>(
```

Comment:
```suggestion
                  make_unique<NotConstraint>(
```
Add a using decl at the beginning of the file.
```
using std::make_unique;
```
This should make the code shorter and a bit easier to read.